apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-user-actions-menu-example.component.html

```diff
@@ -0,0 +1,58 @@
+<fd-shellbar>
+    <fd-shellbar-logo>
+        <a href="#" class="fd-shellbar__logo fd-shellbar__logo--image-replaced" aria-label="SAP"></a>
+    </fd-shellbar-logo>
+    <fd-shellbar-actions>
+        <fd-user-actions-menu [user]="user">
+            <ul fd-list [noBorder]="true" navigationIndicator="true">
+                <li fd-list-item fd-user-actions-menu-item>
+                    <a fd-list-link>
+                        <i fd-list-icon glyph="border"></i>
+                        <span fd-list-title>
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry
```

Comment:
same as above, `title` may be needed to show the entire text in tooltip because it is truncated.